tv_apps/smart-system/js/entry_sheet.js

```diff
@@ -30,6 +30,9 @@ window.EntrySheet = (function invocation() {
 
       // Here we remove entire element once the close button is pressed.
       this.container.removeChild(this.element);
+      if (this.onCloseCallback) {
+        this.onCloseCallback();
```

Comment:
According to the logic, we may call `close` multiple times. Since the execution of this callback is wrapped in transitionend, it should only be called once.

I would suggest to add the following code at the beginning of close function:
```
if (!this.element.classList.contains('active')) {
  // This entry sheet is already closed. No need to re-close it again.
  return;
}
```